examples/tutorials/advanced/contour_map.py

```diff
@@ -33,16 +33,12 @@
 # Contour line settings
 # ---------------------
 #
-# Use the ``annotation`` and ``interval`` parameters to adjust contour line
+# Use the ``annotation`` and ``levels`` parameters to adjust contour line
 # intervals. In the example below, there are contour intervals every 250 meters
 # and annotated contour lines every 1,000 meters.
 
 fig = pygmt.Figure()
-fig.grdcontour(
-    annotation=1000,
-    interval=250,
-    grid=grid,
-)
+fig.grdcontour(annotation=1000, levels=250, grid=grid)
```

Comment:
Can you please update this example to always put `grid` as the first parameter?
```suggestion
fig.grdcontour(grid=grid, annotation=1000, levels=250)
```